.goreleaser.yaml

```diff
@@ -1,31 +1,46 @@
 project_name: azioncli
+
 builds:
   - env:
       - CGO_ENABLED=0
     main: ./cmd/azioncli
     ldflags:
-        - -X github.com/aziontech/azion-cli/pkg/cmd/version.BinVersion={{ .Env.BIN_VERSION }}
-        - -X github.com/aziontech/azion-cli/pkg/constants.AuthURL={{ .Env.AUTH_URL }}
-        - -X github.com/aziontech/azion-cli/pkg/constants.ApiURL={{ .Env.API_URL }}
+      - -X github.com/aziontech/azion-cli/pkg/cmd/version.BinVersion={{ .Env.BIN_VERSION }}
+      - -X github.com/aziontech/azion-cli/pkg/constants.AuthURL={{ .Env.AUTH_URL }}
+      - -X github.com/aziontech/azion-cli/pkg/constants.ApiURL={{ .Env.API_URL }}
+    binary: azioncli
     goos:
       - linux
       - darwin
+
 archives:
   - replacements:
       darwin: Darwin
       linux: Linux
       amd64: x86_64
+  - files:
+      - completions/*
+
+before:
+  hooks:
+    - chmod +x ./scripts/completions.sh
+    - ./scripts/completions.sh
 
 brews:
-- name: azioncli
-  homepage: https://github.com/aziontech/azion-cli
-  description: The developer friendly way to interact with Azion
-  license: "MIT"
-  dependencies:
+  - name: azioncli
```

Comment:
same as above... why change the identation here? did the previous one generate an error for you?

Reply:
I don't know how to answer you, I don't know how this field got there, in my place I ended up not seeing it and it doesn't reflect the same